ClassWork_9/Homework_7/Homework.html

```diff
@@ -0,0 +1,180 @@
+    name : "Women's Dress Sleeveless",
+    description : " Sleeveless Cocktail Dresses till Knee Length",
+    Price: "37€" ,
```

Comment:
price shall start with small p

Reply:
already resolved in my latest hw,please review it